routes/#checkout.js

```diff
@@ -323,6 +324,11 @@ module.exports = (checkoutBody, checkoutRespond, storeId) => {
                       }
                     })
 
+                    if (transaction.amount && (transaction.amount > amount.total)) {
```

Comment:
```suggestion
                    if (transaction.amount > amount.total) {
```

`transaction.amount` sempre é definido, ainda que não fosse `undefined > number` é sempre falso...